### ISSUE TITLE
Implement Rebranding

### ISSUE THREAD

**Issue by brunosan** [datapartnership/datapartnership.org#2]
I just pushed 27f7cea which updates the name in the code. I don't see it deploying (blocked by #1), but also the email remains the same, as we don't yet have a new email. 

Let's update it when it's done, and make sure there are no other leftover branding to fix.

🎉 for the new name! 😄

**Comment by brunosan** [comment]
Since this code is **not** the one deployed on datapartnership.org/ (see https://github.com/datapartnership/datapartnership.org/issues/1#issuecomment-567007783) I've reverted  it on the main branch, and moved it to a new separate branch `rebranding`. Once we are clear on #2 we can deploy it properly.

**Comment by dc-coder** [comment]
correct, this is not the code on the main site.  Holly just added you to the netlify team, so you should be able to see the deployed version there. 

I think we need the following branches:
- develop: until I finish developing the thing
- master: once develop stage is done, we'll move to master
- preview: a temp preview branch that I can setup, in case someone wants to preview the content before hitting publish.
